addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/handler/KodiHandler.java

```diff
@@ -277,27 +314,27 @@ public void updateVolume(int volume) {
     @Override
     public void updatePlayerState(KodiState state) {
         switch (state) {
-            case Play:
-                updateState(CHANNEL_CONTROL, PlayPauseType.PLAY);
-                updateState(CHANNEL_STOP, OnOffType.OFF);
-                break;
-            case Pause:
-                updateState(CHANNEL_CONTROL, PlayPauseType.PAUSE);
-                updateState(CHANNEL_STOP, OnOffType.OFF);
-                break;
-            case Stop:
-            case End:
-                updateState(CHANNEL_CONTROL, PlayPauseType.PAUSE);
-                updateState(CHANNEL_STOP, OnOffType.ON);
-                break;
-            case FastForward:
-                updateState(CHANNEL_CONTROL, RewindFastforwardType.FASTFORWARD);
-                updateState(CHANNEL_STOP, OnOffType.OFF);
-                break;
-            case Rewind:
-                updateState(CHANNEL_CONTROL, RewindFastforwardType.REWIND);
-                updateState(CHANNEL_STOP, OnOffType.OFF);
-                break;
+        case Play:
```

Comment:
Sorry, I only saw the large diff and did not look further into it. Thanks for the formatting.